### PR TITLE
🌱 Release v1.6.0-beta.0

### PR DESCRIPTION
**Pull request by g-gaston**
/area release

### CODE REVIEW

**Review by furkatgofurov7 — COMMENTED** [review]
@g-gaston thank you, looks awesome!

Few nits + any diff of commits the PR is missing:

File: CHANGELOG/v1.6.0-beta.0.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+- e2e: Improve labels/annotations in CAPD test ClusterClass (#9469)
+- e2e: Refactor e2e ownerRef test utils (#9313)
+- e2e: Test/e2e: structure resources by namespace/kind again (#9462)
+- e2e: Use existing value of `SKIP_RESOURCE_CLEANUP` if set in environment (#9152)
+- IPAM: Add age column to kubectl output (#9521)
+- KCP: Controlplane: add a test case for syncMachines where the InfraMachine does not exist. (#8992)
+- KCP: Remove disableInPlacePropagation field in KCP controller (#9099)
+- KCP: Remove redundant GetRESTConfig in KCP Management.GetWorkloadCluster (#9448)
+- logging: Change default log level to 2 (#9093)
+- logging: Fix patch errors not being logged (#9224)
```
Comment (killianmuldoon) [inline CHANGELOG/v1.6.0-beta.0.md:215]:
```suggestion
- Logging: Fix patch errors not being logged (#9224)
```

File: CHANGELOG/v1.6.0-beta.0.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+- logging: Set controller-runtime logger for clusterctl (#9107)
+- MachinePool: Add MachinePool Builders (#9346)
+- MachinePool: Add util function to get MachinePool by label (#9219)
+- metrics: Hack/observability: add capi_machine_status_certificatesexpirydate metric (#9084)
+- metrics: Hack: bump kube-state-metrics and prometheus charts (#9352)
```
[the remark this answers is not in the document]
Comment (killianmuldoon) [reply CHANGELOG/v1.6.0-beta.0.md:220]: Capitalization should be fixed up in the release notes tool

**Review by killianmuldoon — COMMENTED** [review]
Looking good - will take a final look once the commits are squashed.

File: CHANGELOG/v1.6.0-beta.0.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (furkatgofurov7) [reply CHANGELOG/v1.6.0-beta.0.md]: Let's create an issue to not forget it

File: CHANGELOG/v1.6.0-beta.0.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+- ClusterCacheTracker: Fix accessor deletion on health check failure (#9025)
+- ClusterClass: Add ownerRefs to BootstrapConfig/InfraMachinePool in classy Clusters (#9389)
+- ClusterClass: Embed ssa.FilterObjectInput into HelperOption to remove duplication (#9512)
+- ClusterClass: Fix some nits in Cluster topology engine tests (#9464)
+- ClusterClass: Improve Cluster variable defaulting/validation errors (#9452)
+- ClusterClass: Improve message for TopologyReconciledCondition (#9400)
+- ClusterClass: Make ClusterClass generated object names consistent (#9254)
+- ClusterClass: Minor fixes for CC+MP implementation (#9318)
+- clusterctl: Check resource blocking clusterctl move during discovery (#9246)
+- Dependency: Update cert-manager to v1.12.3 (#9197)
```
Comment (furkatgofurov7) [inline CHANGELOG/v1.6.0-beta.0.md:143]: we should probable move this down

Comment (killianmuldoon) [inline CHANGELOG/v1.6.0-beta.0.md:143]: Out of order

**Review by killianmuldoon — COMMENTED** [review]
One last nit

**Review by killianmuldoon — COMMENTED** [review]
Great work!

/approve
/lgtm

Feel free to unhold

**Review by furkatgofurov7 — APPROVED** [review]
Thanks a lot @g-gaston!

/approve
/lgtm

**Pull request merged**